app/models/veteran.rb

```diff
@@ -21,7 +21,7 @@ class Veteran < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
   EMAIL_REGEX = /\A[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]+\z/
-  validates :email, format: { with: EMAIL_REGEX, message: 'Please provide a valid e-mail address' }
+  validates :email, format: { with: EMAIL_REGEX, message: 'must be valid' }
```

Comment:
Right - I'm asking what is the message without defining `message`? I suspect it might be 'must be valid' making defining it superfluous code.